src/order.rs

```diff
@@ -306,8 +358,9 @@ impl Builder {
             slippage,
             post_only,
         } = self;
-        let side = side.ok_or(Error::NoSide)?;
-        let price = price.ok_or(Error::NoPrice)?;
+        let symbol = symbol.ok_or_else(Error::no_symbol)?;
+        let side = side.ok_or_else(Error::no_side)?;
+        let price = price.ok_or_else(Error::no_price)?;
```

Comment:
I realize that the price is not always required. In the case of market or stop orders the price is derived from the orderbook.

Make an issue for this so we can address this in a followup PR.

Reply:
Created issue #19 in 68e4fbe